### ISSUE TITLE
Importance sampling in Greedy GQ

### ISSUE THREAD

**Issue by armahmood** [rlpy/rlpy#5]
First of all, thanks a lot for making this wonderful framework. I started looking at it and feeling really good about it.

I have a question about the Greedy-GQ implementation. I think the implementation you have is basically for lambda=0, and when lambda>0, you have heuristically added the eligibility trace vector. 

However, there is a more principled Greedy GQ(lambda) with eligibility traces in Maei's (2011) thesis which uses importance sampling ratios. I am wondering whether you considered using such ratios in your implementation.

**Comment by chrodan** [comment]
You're right. The lambda>0 implementation is simply a heuristic extension. It would be great to have an implementation of the version proposed in Hamid's thesis. 
If you happen to implement it, please make a pull request!
I am also super curious to see some empirical comparisons between the two versions (the current heuristic works surprisingly well in practice).
